app/Http/Controllers/Maps/DeviceDependencyController.php

```diff
@@ -154,7 +154,8 @@ public function dependencyMap(Request $request)
             $devices_by_id = $this->highlightDevices($devices_by_id, $this->parentDeviceIds);
         }
 
-        array_multisort(array_column($device_list, 'label'), SORT_ASC, $device_list);
```

Comment:
While it seems to work anyway, array_multisort require a variable passed by reference